src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -22,41 +20,38 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-interface OrganizeTabbedLayoutProps {
+interface TabbedLayoutProps {
     fixedHeight?: boolean;
+    title?: string;
+    subtitle?: string;
+    baseHref: string;
+    tabs: { defaultTab?:boolean; href: string; label: string}[];
 }
 
-const OrganizeTabbedLayout: FunctionComponent<OrganizeTabbedLayoutProps> = ({ children, fixedHeight }) => {
+const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({ children, fixedHeight, title, subtitle, tabs, baseHref }) => {
     const intl = useIntl();
     const classes = useStyles();
     const router = useRouter();
-    const { campId, orgId } = router.query;
 
-    const campQuery = useQuery(['campaign', orgId, campId ], getCampaign(orgId  as string, campId as string));
+    let currentTab = router.pathname.split('/').pop();
 
-    const path =  router.pathname.split('/');
-    let currentTab = path.pop();
-
-    if (currentTab === '[campId]' || currentTab === 'campaigns') {
-        currentTab = 'summary';
+    if (router.asPath === baseHref) {
+        currentTab = tabs.find(tab => tab.defaultTab)?.label;
+        if (!currentTab) {
+            console.warn('No default tab selected.');
```

Comment:
I'm not a big fan of these. If not having a default tab is a bug, it should be louder than a warning to developers, and ideally there should be no console output to non-developers.

I would suggest instead throwing an error if `NODE_ENV` is `development`, and else do nothing.

Even better of course would be some sort of compile-time type safety, but I'm not sure how we would achieve that. I guess it would require enums and generics, and probably not be very beautiful. Maybe @djbusstop has an idea?